src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -129,6 +130,12 @@ private DocumentDatabase CreateDocumentsStorage(StringSegment databaseName, Rave
             }
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private void UpdateDatabaseInfoCache(DocumentDatabase database, ServerStore serverStore)
+        {
+            serverStore.DatabaseInfoCache.InsertDatabaseInfo(database.GenerateDatabaseInfo(true));
```

Comment:
Here delete this on load, so we'll only have this for dbs that were successfully shutdown.